experimental/kubernetes/simple/README.md

```diff
@@ -211,3 +211,14 @@ mechanism (Redis), run
 ```
 $ bash scripts/cleanup-spinnaker.sh # This deletes everything with application name spin
 ```
+
+
+
+## Common falls
+
+1. clouddriver not configured properly, check all the logs from `clouddriver` pod, many relate to [this link](http://www.spinnaker.io/docs/troubleshooting-guide#section-my-gke-google-container-engine-cluster-isn-t-showing-up-as-a-kubernetes-account-)
```

Comment:
I wouldn't call this a Clouddriver configuration issue - and it might be helpful to provide more context here (e.g., if you're running on GKE, if your browser console is showing these errors, etc.). You could also provide a link to an issue (e.g., https://github.com/spinnaker/spinnaker/issues/1276), which has a lot of context.